lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -192,11 +192,12 @@ def database_from_yaml
         def database_yaml
           return nil unless File.exist?('config/database.yml')
 
-          yaml = if YAML.respond_to?(:unsafe_load_file)
-                   YAML.unsafe_load_file('config/database.yml')
```

Comment:
I do not remember how I reproduced it before, but reverted this PR changes and it works. 
I think, the user just forgot `default: &default` section in the `database.yml`, because in this case it raises the exact same error as in the issue.